Exercise_2/css/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+    display: flex;
+    flex-direction: column;
+    font-size: 10px;
+    margin: 0;
+}
+
+/*HEADER*/
+.header {
+    display: flex;
+    height: 40vh;
```

Comment:
It can be a good idea to use `vh` for sizing, but you should not forget about minimal height because if you squeeze you browser vertically you will see not a really nice picture.